src/vqv.js

```diff
@@ -16,6 +16,14 @@
       #VQV!'
 */
 
-const vqv = (nome, idade) => {};
+const vqv = (nome, idade) => {
+  if ((nome === undefined) || (idade === undefined)) {
```

Comment:
Gostei muito do seu código 👏
Mas, tem algum motivo especifico para utilizar essa redundancia de parenteses?
```suggestion
  if (nome === undefined || idade === undefined) {
```